GiniVision/Classes/Core/Screens/Camera/Camera.swift

```diff
@@ -34,9 +36,19 @@ final class Camera: NSObject, CameraProtocol {
     var stillImageOutput: AVCaptureStillImageOutput?
     var didDetectQR: ((GiniQRCodeDocument) -> Void)?
     var giniConfiguration: GiniConfiguration
+    
+    lazy var isFlashSupported: Bool = {
+        #if targetEnvironment(simulator)
```

Comment:
But the simulator doesn't support flash. This default made sense for testing the feature, but now that it's done, maybe not so much.

Reply:
The camera button also doesn't make sense and is still there, for testing purposes makes sense.